frontend/test/currentUser/currentUser.store.spec.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { CurrentUserActions } from '@/v5/store/currentUser/currentUser.redux';
+import {
+	selectPersonalDataIsUpdating,
+	selectApiKeyIsUpdating,
+	selectCurrentUser,
+	selectFirstName,
+} from '@/v5/store/currentUser/currentUser.selectors';
+import { currentUserMockFactory, generatePersonlData } from './currentUser.fixtures';
+import { createTestStore } from '../test.helpers';
+
+describe('CurrentUser: store', () => {
+	let dispatch, getState;
+
+	beforeEach(() => {
+		({ dispatch, getState} = createTestStore());
+	})
+
+	it('should set currentUser', () => {
```

Comment:
can we add a case of selecting the current user without setting it ? just to guarantee the selector doesnt break

Reply:
Sure, but just so you know, I think the selector uses a default value `|| {}` which is never used given the `INITIAL_STATE` set for currentUser
![image](https://user-images.githubusercontent.com/43547703/185355273-6ec84a3f-76f1-4f19-97af-ecffd0a6f989.png)

currentUser starts as a non-empty object which properties are empty strings

Reply:
Yea, I know. I just wanted to be sure that the selector wont break